lbrynet/core/Peer.py

```diff
@@ -12,8 +12,7 @@ def __init__(self, host, port):
         self.stats = defaultdict(float)  # {string stat_type, float count}
 
     def is_available(self):
-        if (self.attempt_connection_at is None or
-            datetime.datetime.today() > self.attempt_connection_at):
+        if self.attempt_connection_at is None or datetime.datetime.today() > self.attempt_connection_at:
```

Comment:
use `utils.now()` (from lbrynet.core.utils`) instead of datetime.datetime.today as it will make mocking easier when testing.

Can also be
```
return (
    self.attempt_connection_at is None or
    utils.now() > self.attempt_connection_at
)
```